src/core/ext/xds/xds_common_types.h

```diff
@@ -36,7 +36,7 @@ struct Duration {
   int64_t seconds = 0;
   int32_t nanos = 0;
 
-  Duration() = default;
+  Duration() : seconds(0), nanos(0) {}
```

Comment:
Drive-by comment: it seems the code here is recent, not in 1.42. Maybe just `Duration() {}` since seconds and nanos are already default initialized?

Reply:
There is no constructor for Duration on v1.42.0. But I have test using `Duration() = default` still has the problem of #28147 .
I have tested `Duration() : seconds(0), nanos(0) {}` in my case and it works. But I'm not sure if it will produce the correct codes when using `Duration(){}` with initializing members in declaration.
I need more suggestion here and thanks.